core/spec/lib/spree/core/controller_helpers/order_spec.rb

```diff
@@ -74,6 +75,23 @@ class FakesController < ApplicationController
         }.from(nil).to("0.0.0.0")
       end
     end
+
+    context 'build_order_if_necessary option is true' do
+      subject { controller.current_order(build_order_if_necessary: true) }
+
+      it 'creates new order' do
```

Comment:
since the order is not persisted, IMO this should read `it builds a new order` instead.